app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/InsuranceClaim.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BcGov.Fams3.SearchApi.Contracts.Person
+{
+    public  class InsuranceClaim : PersonalInfo
+    {
+
+        [Description("Claim Type")]
+        public string ClaimType { get; set; }
+
+        [Description("Claim Status")]
+        public string ClaimStatus { get; set; }
+
+        [Description("claim number")]
+        public string ClaimNumber { get; set; }
+
+        [Description("claim center")]
+        public string ClaimCenter { get; set; }
```

Comment:
claimCenter is a string?

Reply:
We need to remove this and wait for feedback from Locate Services in the weekly checkin